backend/src/services/secret/secret-queue.ts

```diff
@@ -562,6 +562,30 @@ export const secretQueueFactory = ({
       throw new Error("Secret path not found");
     }
 
+    const sendIntegrationSyncFailedMail = async (integrationId: string, syncMessage: string | null) => {
+      const appCfg = getConfig();
+
+      // If smtp is not configured, we can return early without having to fetch the project members.
+      if (!appCfg.isSmtpConfigured) return;
+
+      const projectMembers = await projectMembershipDAL.findAllProjectMembers(projectId);
```

Comment:
Should all the members get it or just admins?

Also i think we may need a  debounced check to ensure a lot of mails are not send when all the secret sync fails. Because from dashboard each change triggers sync integration.

Now consider integration failing and I did a quitie some change, I would be bombarded with mail